app/components/post_attachment_opengraph/post_attachment_opengraph.js

```diff
@@ -34,7 +34,9 @@ export default class PostAttachmentOpenGraph extends PureComponent {
     constructor(props) {
         super(props);
 
-        this.state = this.getBestImageUrl(props.openGraphData);
+        this.state = {
```

Comment:
not really sure why the deconstruct here, but not that it matters much

Reply:
This was the reason: https://github.com/mattermost/mattermost-mobile/pull/4270/commits/b661b074c4d64f9cd81dfb91f7544634cc1243bd#diff-668888672f6dcf875dbc9d52809cb5dfR55, but again, need to be cleaned up from all the changes spawn from this review process.